static/elements/chromedash-activity-log.js

```diff
@@ -128,7 +128,7 @@ export class ChromedashActivityLog extends LitElement {
 
   // Display how long ago the comment was created compared to now.
   formatCommentRelativeDate(comment) {
-    const commentDate = new Date(`${comment.created} UTC`);
+    const commentDate = new Date(`${comment.created}Z`);
```

Comment:
Does that actually work?
When I look at the value of a comment's created field in devtools, I see a string like
 "2022-01-07 20:21:06.334990"
but the format expected is this
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date
I was wrong when I said add a "Z", that is optional.
But, it does require a "T" in the middle.

Reply:
If you write a unit test for this, our `npm run wettest` target should execute the tests on both chrome and firefox, so we'll know for sure if it works.

Reply:
I wasn't sure if this change would work, but it does fix the issue on Chrome and Firefox. However, I went back and checked after your comment and it does not fix the issue on Safari, which I should have checked (I saw Firefox and Chrome working and assumed the issue was fixed).

Replacing the space with a "T" does fix the issue in all browsers as far as I can tell, and I'll write a test as well.